docs/user/production-considerations/security-production-considerations.asciidoc

```diff
@@ -0,0 +1,52 @@
+[role="xpack"]
+[[Security-production-considerations]]
+== Security production considerations
+
+++++
+<titleabbrev>Security</titleabbrev>
+++++
+:keywords: administrator, analyst, concept, setup, security
+:description: Consider the production components for Kibana security.
+
+[float]
+[[configuring-kibana-shield]]
+=== Use {stack} {security-features}
+
+You can use {stack} {security-features} to control what {es} data users can
+access through Kibana.
+
+When {security-features} are enabled, Kibana users have to log in. They need to
+have a role granting <<kibana-privileges, Kibana privileges>> as well as access
+to the indices they will be working with in Kibana.
+
+If a user loads a Kibana dashboard that accesses data in an index that they
+are not authorized to view, they get an error that indicates the index does
+not exist.
+
+For more information on granting access to Kibana, see <<xpack-security-authorization>>.
+
+[float]
+[[csp-strict-mode]]
+=== Require Content Security Policy
+
+Kibana uses a Content Security Policy to help prevent the browser from allowing
+unsafe scripting, but older browsers will silently ignore this policy. If your
+organization does not need to support Internet Explorer 11 or much older
+versions of our other supported browsers, we recommend that you enable Kibana's
+`strict` mode for content security policy, which will block access to Kibana
+for any browser that does not enforce even a rudimentary set of CSP
+protections.
```

Comment:
We [stopped supporting IE in Kibana 7.9](https://www.elastic.co/support/matrix#matrix_browsers), so this is stale info.
I did check our other supported browsers (Chrome, Firefox, Safari) -- each of them first started supporting Content Security Policy around 2012-2013. So I suppose we should leave this section in here and just remove the bit about IE, yeah?